.github/workflows/test.yml

```diff
@@ -0,0 +1,100 @@
+name: Run test on linux mac
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ '*' ]
+  pull_request:
+
+env:
+  BUILD_DIR: build
+  BUILD_TREADS: 2
+  CTEST_OUTPUT_ON_FAILURE: 1
+  GITHUB_HUNTER_USERNAME: ${{ secrets.HUNTER_USERNAME }}
+  GITHUB_HUNTER_TOKEN: ${{ secrets.HUNTER_TOKEN }}
+  INDOCKER_IMAGE: soramitsu/kagome-dev:8
+
+jobs:
+  MacOS:
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: install
+      run: ./housekeeping/macos/dependency.sh
+    - name: build
+      env:
+        BUILD_TREADS: 4
+        DEVELOPER_DIR: /Applications/Xcode_11.app/Contents/Developer
+      run: ./housekeeping/makeBuild.sh  -DCOVERAGE=OFF -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain/cxx17.cmake
```

Comment:
Can we move the build option (like '-DCOVERAGE=OFF....') to a environment variable to have same style

Reply:
I see some problem, cmake expect multiple argument but if we put it in env variable  `BUILD_CMAKE_OPTION='-DCOVERAGE=OFF -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain/cxx17.cmake' `
executed command:
`cmake . -B${BUILD_DIR} ${BUILD_CMAKE_OPTION}` 
will be parsed as 
`cmake . -Bbuild 'some text with space'`
which create error: `unknown argument: "'some text with space'"`

In theory we can set without quotas  
`BUILD_CMAKE_OPTION=-DCOVERAGE=OFF\ -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain/cxx17.cmake `
but this solution is not universal.